test/python/test_mr_conn.py

```diff
@@ -0,0 +1,26 @@
+from model_registry import ModelRegistry
+
+
+def main(server: str, port: int):
+    mr = ModelRegistry(server, port, author="test", is_secure=False)
+
+    mr.register_model(
+        "my-model",
+        "https://mybucket.uri/",
+        version="2.0.0",
+        model_format_name="onnx",
+        model_format_version="1",
+        storage_key="my-data-connection",
+        storage_path="path/to/model",
+        metadata={
+            "day": 1,
+            "split": "train",
+        },
+    )
+    print(mr.get_registered_model("my-model"))
```

Comment:
shouldn't this be an assert of some kind?

Reply:
we can do that for sure. I just thought it'd be meaningless considering we're already testing that extensively on the unit tests -- so this one can only fail if the calls don't complete correctly. Either way we can double cover.